openupgrade_scripts/scripts/iap/18.0.1.1/upgrade_analysis.txt

```diff
@@ -14,11 +14,11 @@ iap          / iap.account              / state (selection)             : NEW se
 iap          / iap.account              / warning_threshold (float)     : is now stored
 iap          / iap.account              / warning_threshold (float)     : not related anymore
 iap          / iap.account              / warning_user_ids (many2many)  : NEW relation: res.users
-iap          / iap.service              / description (char)            : NEW required
+iap          / iap.service              / description (char)            : NEW required, translate
```

Comment:
the type change should be fine, the orm handles it

Reply:
That's not a change in Odoo, but https://github.com/OCA/server-tools/pull/3299